curriculum/challenges/english/25-front-end-development/quiz-html-accessibility/66ed9026f45ce3ece4053ebb.md

```diff
@@ -17,439 +17,446 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+Accessibility focuses on making websites faster to load.
 
 ---
 
-Placeholder distractor 2
+Accessibility is a belief to make websites compatible with older browsers.
 
 ---
 
-Placeholder distractor 3
+Accessibility is a type of coding style.
 
 #### --answer--
 
-Placeholder answer
+Accessibility is a set of standardized practices that ensure online content can be viewed by users with disabilities.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the best practice for creating accessible headings?
 
 #### --distractors--
 
-Placeholder distractor 1
+Add SEO-friendly keywords 
 
 ---
 
-Placeholder distractor 2
+Create humorous headings 
 
 ---
 
-Placeholder distractor 3
+Place `h6` headings at the beginning of your webpage 
 
 #### --answer--
 
-Placeholder answer
+Put headings in chronological order 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What are the six main categories of ARIA roles?
 
 #### --distractors--
 
-Placeholder distractor 1
+Document Structure, Landmark, Window, Explicit, Fidget, and Footer
 
 ---
 
-Placeholder distractor 2
+Landmark, House, Live Region, Application, Window, and Extension
 
 ---
 
-Placeholder distractor 3
+Window, Alertdialog, Article, Cell, Banner, and Button
 
 #### --answer--
 
-Placeholder answer
+Document Structure, Landmark, Window, Abstract, Widget, and Live Role 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the correct way to add an `alt` attribute to an `<img>` tag?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+alt="cat running"<img src="cat.jpg"> 
+```
 
 ---
 
-Placeholder distractor 2
+```html
+alt=""<img src="cat.jpg"> 
+```
 
 ---
 
-Placeholder distractor 3
+```html
+<img src="cat.jpg"> alt="cat running"
+```
 
 #### --answer--
 
-Placeholder answer
+```html
+<img src="cat.jpg" alt="cat running"> 
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is WAI-ARIA?
 
 #### --distractors--
 
-Placeholder distractor 1
+WAI-ARIA is a programming language specifically designed for creating animated web graphics
 
 ---
 
-Placeholder distractor 2
+WAI-ARIA is a set of guidelines for making websites more visually appealing and colorful
 
 ---
 
-Placeholder distractor 3
+WAI-ARIA is a tool for optimizing website loading speeds and server performance.
 
 #### --answer--
 
-Placeholder answer
+WAI-ARIA is a set of standardized practices that ensure online content can be viewed by users with disabilities.
```

Comment:
```suggestion
WAI-ARIA is a set of standardized practices that helps make online content more accessible to people with disabilities.
```

Another nitpick. It is hard to "ensure" that content will be accessible to everyone. The best we can do is get as close as possible.